runtime/mozart/Cargo.toml

```diff
@@ -0,0 +1,336 @@
+grandpa_primitives = { workspace = true }
+inherents = { workspace = true }
+mozart-runtime-constants = { workspace = true }
+offchain-primitives = { workspace = true }
+sp-api = { workspace = true }
+sp-arithmetic = { workspace = true }
+sp-core = { workspace = true }
+sp-genesis-builder = { workspace = true }
+sp-io = { workspace = true }
+sp-mmr-primitives = { workspace = true }
+sp-runtime = { workspace = true }
+sp-session = { workspace = true }
+sp-staking = { workspace = true }
+sp-std = { workspace = true }
+sp-storage = { workspace = true }
+sp-version = { workspace = true }
+tx-pool-api = { workspace = true }
+
+frame-executive = { workspace = true }
+frame-support = { workspace = true, features = [ "tuples-96" ] }
+frame-system = { workspace = true }
+frame-system-rpc-runtime-api = { workspace = true }
+pallet-asset-rate = { workspace = true }
+pallet-authority-discovery = { workspace = true }
+pallet-authorship = { workspace = true }
+pallet-babe = { workspace = true }
+pallet-balances = { workspace = true }
+pallet-beefy = { workspace = true }
+pallet-beefy-mmr = { workspace = true }
+pallet-bounties = { workspace = true }
+pallet-child-bounties = { workspace = true }
+pallet-collective = { workspace = true }
+pallet-conviction-voting = { workspace = true }
+pallet-democracy = { workspace = true }
+pallet-elections-phragmen = { workspace = true }
+pallet-grandpa = { workspace = true }
+pallet-identity = { workspace = true }
+pallet-indices = { workspace = true }
+pallet-membership = { workspace = true }
+pallet-message-queue = { workspace = true }
+pallet-mmr = { workspace = true }
+pallet-multisig = { workspace = true }
+pallet-nis = { workspace = true }
+pallet-offences = { workspace = true }
+pallet-parameters = { workspace = true }
+pallet-preimage = { workspace = true }
+pallet-proxy = { workspace = true }
+pallet-ranked-collective = { workspace = true }
+pallet-recovery = { workspace = true }
+pallet-referenda = { workspace = true }
+pallet-root-testing = { workspace = true }
+pallet-scheduler = { workspace = true }
+pallet-session = { workspace = true }
+pallet-society = { workspace = true }
+pallet-staking = { workspace = true }
+pallet-state-trie-migration = { workspace = true }
+pallet-sudo = { workspace = true }
+pallet-timestamp = { workspace = true }
+pallet-tips = { workspace = true }
+pallet-transaction-payment = { workspace = true }
+pallet-transaction-payment-rpc-runtime-api = { workspace = true }
+pallet-treasury = { workspace = true }
+pallet-utility = { workspace = true }
+pallet-vesting = { workspace = true }
+pallet-whitelist = { workspace = true }
+pallet-xcm = { workspace = true }
+pallet-xcm-benchmarks = { workspace = true, optional = true }
+
+frame-benchmarking = { workspace = true, optional = true }
+frame-system-benchmarking = { workspace = true, optional = true }
+frame-try-runtime = { workspace = true, optional = true }
+hex-literal = { version = "0.4.1" } # FIXME: not same as workspace
```

Comment:
`rococo` uses `hex-literal = { version = "0.4.1" }` but `tanssi` workspace `Cargo.toml` uses `hex-literal = { version = "0.3.4" }` so they are a minor version apart (and thus `Cargo.lock` defines it twice). 

Therefore, for now I've left the version defined in the new crate for now as it's unclear best path to take

Reply:
but does something happen if you point to 0.3.4?

Reply:
I can try and then run full suite of tests - I will also check the `Changelog`.

Reply:
I guess my statement was more is it preferred to bring both down to `0.3.4` or both up to `0.4.1` - which you've answered, so I will test with that

Reply:
Unit tests ran fine